web/client/components/geostory/builder/SectionsPreview.jsx

```diff
@@ -0,0 +1,119 @@
+ * @param {object} [content={}] content that have to render preview
+ */
+const Preview = ({ width } = {}) => {
+    const res = 1080 / 1920;
+    const height = width * res;
+    return (
+        <div style={{ position: 'relative', width, height }}>
+            <div style={{ position: 'absolute', width: '100%', height: '100%', backgroundColor: '#dddddd', display: 'flex' }}>
+                <Glyphicon glyph="1-map" style={{ fontSize: height / 2, margin: 'auto' }} />
+            </div>
+        </div>
+    );
+};
+
+/**
+ * Transforms a geostory section into a SideGrid item.
+ * @param {object} section the section to transform
+ */
+const sectionToItem = ({
+    contents,
+    type,
+    title,
+    id
+}) => {
+    return {
+        id: id,
+        preview: <Icon type={type} />,
+        tools: <Toolbar
+            btnDefaultProps={{
+                className: 'square-button-md no-border'
+            }}
+            buttons={[
+                {
+                    glyph: 'zoom-to',
+                    visible: contents.length === 1,
+                    tooltip: 'Zoom to content'
+                }
+            ]} />,
+        title: title,
+        description: `type: ${type}`,
+
+        body: contents
+            ? <div style={{ position: 'relative' }}>
+                <SideGrid
+                    containerId={id}
+                    isDraggable
+                    cardComponent={SideCard}
+                    items={contents.map((content) => {
+                        const foreground = content && content.foreground || {};
+                        return {
+                            id: content.id,
+                            preview: <Icon type={content.type + (foreground.textContainerPosition || '')} />,
+                            tools: <Toolbar
+                                btnDefaultProps={{
+                                    className: 'square-button-md no-border'
+                                }}
+                                buttons={[
+                                    {
+                                        glyph: 'zoom-to'
+                                    }
+                                ]} />,
+                            title: capitalize(content.type),
+                            description: `type: ${content.type}`,
+                            body: (
+                                <ContainerDimensions>
+                                    <Preview content={content} />
+                                </ContainerDimensions>
+                            )
+                        };
+                    })} />
+            </div>
+            : <ContainerDimensions>
+                <Preview content={contents[0]} />
+            </ContainerDimensions>
+    };
+
+};
+
+/**
+ * Shows a preview list of the slides/sections of a story
+ * @SectionsPreview
+ * @param {object[]} [sections=[]] Array of sections to display
+ */
+export default ({ sections = [] }) => (<SideGrid
```

Comment:
SideGrid needs some style adjustment at application level, in particular for margin and issue inherit from .row and .col styles from bootstrap